tests/equilibria/fem_fixed_boundary/test_fem_magnetostatic_2D.py

```diff
@@ -99,7 +100,9 @@ def test_simple_thin_coil(self):
             subdomains=True,
         )
         dolfin.plot(mesh)
-        plt.show()
+
+        if tests.PLOTTING:
```

Comment:
If we could move `dolfin.plot(mesh)` into this if block too, that would be better

Reply:
done